lib/common/api/deprecate.ts

```diff
@@ -137,7 +135,7 @@ const deprecate = {
     // inject it and warn about it
     if ((oldName in o) && !(newName in o)) {
       warn()
-      o[newName] = o[oldName]
+      o[newName] = (o as any)[oldName]
```

Comment:
This bothers me – and while trying to fix it, I noticed that we don't use `renameProperty` anywhere. Should we cut it?

Reply:
We don't use it at the moment, but we may in the future and it sucks having to add back in code that we removed.  Better to have a battle tested un-used bit of code than write a new one each time IMO

Reply:
@felixrieseberg if you'd prefer to note lack of use, we can comment it out but i'd rather it remain  there in some form as opposed to removed i think!